webpack.config.test.js

```diff
@@ -1,17 +1,10 @@
-const HtmlWebpackPlugin = require('html-webpack-plugin');
+const HtmlBundlerPlugin = require('html-bundler-webpack-plugin');
 const path = require('node:path');
-const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const RemoveEmptyScriptsPlugin = require('webpack-remove-empty-scripts');
 
 module.exports = {
     mode: 'development',
-    entry: {
-        index: path.join(__dirname, 'examples/src/index.jsx'),
-        style: path.join(__dirname, 'examples/src/scss/style.scss'),
-    },
     output: {
         path: path.join(__dirname, 'examples/dist'),
-        filename: '[name].js',
```

Comment:
You can specify JS output filename in the `js.filename` option of the HtmlBundlerPlugin.